src/firefly/html/lsst-pdac-triview.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <link rel="icon" type="image/x-icon" href="images/fftools-logo-16x16.png">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <link rel="apple-touch-icon" href="images/fftools-logo-57x57_ipad.png"/>
+    <link rel=”apple-touch-startup-image” href=”images/fftools-ipad_splash_768x1004.png”>
+    <title>IRSA Viewer</title>
+
+    <script>
+        window.firefly = {
+            app: {
+                menu: [
+                    {label:'LSST Data', action:'LsstCatalogDropDown'},
+                    {label:'External Images', action:'ImageSelectDropDownCmd'},
+                    {label:'External Catalogs', action:'IrsaCatalogDropDown'},
+                    {label:'Add Chart', action:'ChartSelectDropDownCmd'},
+                    //{label:'Help', action:'app_data.helpLoad', type:'COMMAND'},
+                    ]
+            },
+            MenuItemKeys: {maskOverlay:true},
+            imageTabs: [ 'fileUpload', 'url', '2mass', 'wise', 'sdss', 'msx', 'dss', 'iras' ],
+            irsaCatalogFilter: 'lsstFilter',
+            catalogSpacialOp: 'polygonWhenPlotExist'
```

Comment:
Yes, I agree.  I am thinking about maybe putting them into an object in the store.  Then the rest of the app references them from the store instead of window.firefly.